### PR TITLE
feat: [ASL-4566] PIL-E course purpose HE or training - Part 2

### PR DESCRIPTION
**Pull request by kiran-varma-home-office**

### CODE REVIEW

**Review by mohomeoffice — COMMENTED** [review]
@kiran-varma-home-office  I have added a comment regarding this name change on other pull request

**Review by jeff-horton-ho-sas — APPROVED** [review]

File: schema/training-course.js [hunk 1]
```diff
@@ -21,7 +22,7 @@ class TrainingCourse extends BaseModel {
         },
         startDate: { type: ['string', 'null'], pattern: date.yearMonthDay },
         title: { type: 'string' },
-        coursePurpose: { type: 'string', enum: ['higher-education', 'training'] },
+        coursePurpose: { type: 'string', enum: [trainingCoursePurpose.higherEducation, trainingCoursePurpose.training] },
```
Comment (jeff-horton-ho-sas) [inline schema/training-course.js:25]:
```suggestion
        coursePurpose: { type: 'string', enum: Object.keys(trainingCoursePurpose) },
```

Undo change to asl-metrics

**Comment by parvinba** [comment]
Build are failing..

**Comment by jeff-horton-ho-sas** [comment]
> Build are failing..

That will be fixed when asl-constants is merged and updated.

**Pull request merged**